meta-resin-common/recipes-containers/docker-disk/files/Dockerfile

```diff
@@ -1,9 +1,5 @@
 FROM docker:17.03-dind
-
-RUN apk add --update bash util-linux shadow && rm -rf /var/cache/apk/*
-
+RUN apk add --update util-linux shadow && rm -rf /var/cache/apk/*
```

Comment:
why use `--update` instead of `--no-cache`?

Reply:
This PR doesn't change that. It would be something to address separately.

Reply:
:+1: